.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
![image](https://github.com/momo-87/math-magicians/assets/93039359/26c13b54-8b48-4b89-bb52-579bea3e46c7)

- [ ] It would be nice if you fix the highlighted links in the image to make your readme descriptive enough 👍 because right now they are broken 😨